cmd/sonobuoy/app/cp.go

```diff
@@ -37,15 +43,64 @@ func init() {
 		&path, "path", "./",
 		"TBD: location to output",
 	)
-	// TODO: Other Options.?.?
+	cmd.Flags().String("output", "./archive", "directory to write results")
 	RootCmd.AddCommand(cmd)
 }
 
 func copyResults(cmd *cobra.Command, args []string) {
-	code := 0
-	if err := ops.CopyResults(path); err != nil {
-		errlog.LogError(errors.Wrap(err, "error attempting to copy sonobuoy results"))
-		code = 1
+	outDir, err := cmd.Flags().GetString("output")
+	if err != nil {
+		errlog.LogError(fmt.Errorf("failed to get output flag value: %v", err))
+		os.Exit(1)
+	}
+
+	// Find and load the kubeconfig.
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
```

Comment:
this is definitely gonna need to be factored out. Do you wanna do it or leave it up to me? :p

Reply:
The original config code had this if/elsed

Reply:
we can both leave it and add a ticket to factor it out.

Reply:
@timothysc the original loader has homegrown kubeconfig finder (just binding the `kubeconfig` env var) and doesn't look in the default place of the homeDir unless I'm totally missing something. This uses the same loading rules that kubectl uses, so the experience should be what users already expect.

Reply:
(`NewDefaultClientConfigLoadingRules()` also takes windows into account)